engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -122,6 +122,12 @@
     private final ShaderManager shaderManager;
     private final Camera playerCamera;
 
+    /*
+    * presumably, the eye height should be context.get(Config.class).getPlayer().getEyeHeight() above the ground plane.
+    * It's not, so for now, we use this factor to adjust for the disparity.
+     */
+    private final float GROUND_PLANE_HEIGHT_DISPARITY = -0.7f;
```

Comment:
What I'm finding is this. Presumably using `-context.get(Config.class).getPlayer().getEyeHeight()` should result in the ground plane matching between the actual room and the world, right? But I'm finding that's not the case. If I just use that value, then a controller that is set on the ground hovers 0.7 meters off of the ground.

I'm stumped as to why this could be, but there's something fishy going on if you think about it (or my understanding is flawed). 1 world unit is 1 block high, right? Presumably 1 meter? So 0.7 world units would be less than the height of a single block. But the default eye height in a new game seems to be about 1.5 blocks high, which is consistent with my observations. Anyway, 1.5 meters makes more sense in terms of real-world units too - 1.5 meters is a more realistic height.